Block/Catalog/Product/View.php

```diff
@@ -120,7 +141,7 @@ private function getPlans()
      */
     private function isEnabledBadge($installments_count)
     {
-        return in_array($installments_count, array(2,3,4,10,12));
+        return in_array($installments_count, array(1,2,3,4,10,12));
```

Comment:
Maybe it could be removed (this method was to filter `pay later` that was not manager in badge v1.

If we want be more generic (and works with p7x for example) we must remove this method and its usages ;)

Reply:
Ok - i will remove it for the next patch.